Orange/widgets/unsupervised/owmds.py

```diff
@@ -451,7 +451,7 @@ def _clear_plot(self):
             self._legend_item = None
 
     def update_controls(self):
-        if self.data is None and getattr(self.matrix, 'axis', 1) == 0:
+        if self.data is None and getattr(self.matrix, 'axis', 1) in [0, 1]:
             # Column-wise distances
             attr = "Attribute names"
             self.labelvar_model[:] = ["No labels", attr]
```

Comment:
... and if it's 1, the distance is computed over rows, so the widget shouldn't offer *Attribute names* as labels. The option should be called something like "Index". Offer it only for labels, but not for colors, shapes and sizes since it makes no sense there.

While your at it, fix old typeo: solor -> color.